cluster.go

```diff
@@ -2,6 +2,29 @@
 // Use of this source code is governed by a BSD-style
```

Comment:
Are these notices left in the files intentionally?

Reply:
nope.  removed. (and copyright back to 2012 on those files that had this.)